src/Microsoft.AspNetCore.BuildTools.Tasks/Microsoft.AspNetCore.BuildTools.Tasks.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <!--
+      The netstandard1.0 TFM doesn't actually compile tasks.
+      It's just there so Internal.AspNetCore.Sdk gets a dependency to this project.
+    -->
+    <TargetFrameworks>netstandard1.0;netstandard1.3</TargetFrameworks>
+    <NuspecFile>$(MSBuildThisFileDirectory)$(MSBuildProjectName).nuspec</NuspecFile>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <RootNamespace>Microsoft.AspNetCore.BuildTools</RootNamespace>
+    <Description>MSBuild tasks. This package is intended for Microsoft use only</Description>
+    <EnableDefaultItems Condition="'$(TargetFramework)' == 'netstandard1.0'">false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NETStandard.Library" Version="1.6.0" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <PackageReference Include="Microsoft.Build.Framework" Version="15.1.0-preview-000523-01" PrivateAssets="All" />
```

Comment:
No, they can float. Although, it doesn't really matter. MSBuild will lift the task to whatever version it carries.